content_schemas/formats/worldwide_office.jsonnet

```diff
@@ -0,0 +1,20 @@
+(import "shared/default_format.jsonnet") + {
+  definitions: (import "shared/definitions/_whitehall.jsonnet") + {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        access_and_opening_times: {
+           type: [
+             "string",
+             "null",
```

Comment:
I _think_ we might not need to specify `null` here, since the field isn't being marked as optional.  Maybe that could be tested by having two examples: one with `access_and_opening_times` and one without?

Reply:
We have examples of both:
- [Required properties with null specified](https://github.com/alphagov/publishing-api/blob/067cd760d76e91d97b47d5e8edecff2ac0fe5ea7/content_schemas/dist/formats/answer/notification/schema.json#L562)
- [Optional properties with null specified](https://github.com/alphagov/publishing-api/blob/067cd760d76e91d97b47d5e8edecff2ac0fe5ea7/content_schemas/formats/shared/default_properties/publisher.jsonnet#L12)

I think this is right, `access_and_opening_times` is currently optional (because it's not in the `required` array), and we _do_ want to be able to provide a `null` value here (which means it needs to be [optional](https://json-schema.org/understanding-json-schema/reference/object.html#required:~:text=In%20JSON%20a%20property%20with%20value%20null%20is%20not%20equivalent%20to%20the%20property%20not%20being%20present.%20This%20fails%20because%20null%20is%20not%20of%20type%20%E2%80%9Cstring%E2%80%9D%2C%20it%E2%80%99s%20of%20type%20%E2%80%9Cnull%E2%80%9D) with a `null` type)

Reply:
Perfect, thanks for confirming!